guides/common/modules/proc_assigning-content-view-environments-to-hosts-by-using-subscription-manager.adoc

```diff
@@ -0,0 +1,38 @@
+[id="assigning-content-view-environments-to-hosts-by-using-subscription-manager"]
+= Assigning content view environments to hosts by using Subscription Manager
+
+To assign content view environments to hosts by using Subscription Manager, perform the following steps:
+
+.CLI procedure
+. Run the `subscription-manager environments --list` command on a host to display the available environments and identify which ones to pass in:
```

Comment:
```suggestion
. Display the available environments and identify which ones to pass in:
```
Don't duplicate the command in the step.

Please, fix this for **all steps in all modules**.